sources/packages/api/src/route-controllers/program-info-request/program-info-request.controller.ts

```diff
@@ -360,7 +361,10 @@ export class ProgramInfoRequestController extends BaseController {
         ].includes(error.name)
       ) {
         throw new UnprocessableEntityException(
-          `${error.name} ${error.message}`,
+          new ApiProcessError(
+            `${error.name} ${error.message}`,
```

Comment:
error.message is alone sufficient here. not required to concatenate with error.name.